src/NextGen/DonationForm/DataTransferObjects/LegacyPurchaseFormData.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace Give\NextGen\DonationForm\DataTransferObjects;
+
+use Give\Donations\Models\Donation;
+use Give\Donors\Models\Donor;
+
+/**
+ * @unreleased
+ */
+class LegacyPurchaseFormData
```

Comment:
Can you please update the docblock to explain its intended use?

Reply:
👍 f783a3fa